c2corg_api/views/user.py

```diff
@@ -34,6 +34,15 @@
 MINIMUM_PASSWORD_LENGTH = 3
 
 
+def valid_email(email):
```

Comment:
> I'm afraid it will be conflicting with my pending PR #1390 though :P

It does conflict as such, but I don't think that they are incompatible, do they?
Ho would you rather proceed? Which PR should we merge to master first?